homeassistant/components/lutron_caseta/light.py

```diff
@@ -48,37 +53,145 @@ async def async_setup_entry(
 
 
 class LutronCasetaLight(LutronCasetaDeviceUpdatableEntity, LightEntity):
-    """Representation of a Lutron Light, including dimmable."""
+    """Representation of a Lutron Light, including dimmable, white tune, and spectrum tune."""
+
+    def __init__(self, light, data) -> None:
+        """Initialize the light and set the supported color modes."""
+        super().__init__(light, data)
+        if light["type"] == "SpectrumTune":
+            self._attr_supported_color_modes = {
+                ColorMode.HS,
+                ColorMode.COLOR_TEMP,
+                ColorMode.BRIGHTNESS,
+                ColorMode.WHITE,
+            }
+        elif light["type"] == "WhiteTune":
+            self._attr_supported_color_modes = {
+                ColorMode.COLOR_TEMP,
+                ColorMode.BRIGHTNESS,
+                ColorMode.WHITE,
```

Comment:
It doesn't look like WHITE should be set here
https://developers.home-assistant.io/docs/core/entity/light?_highlight=light#color-modes

> If this mode is supported, the light must also support at least one of ColorMode.HS, ColorMode.RGB, ColorMode.RGBW, ColorMode.RGBWW or ColorMode.XY.

Reply:
Hmm thats odd, Is there someone I could ask about that? it seems to be working just fine when I try it, and I don't know why it wouldn't be able to support ColorMode.white and ColorMode.COLOR_TEMP (See the W icon next to the color picker)

Enabling white mode (warm dim) is a large part of the lutron lightstrips, so I would be curious to make sure there is a reason it's not supported, or just forgot to be added to the documentation in the list.

![image](https://github.com/home-assistant/core/assets/3183738/51acb115-d278-47ef-807e-185a3a174d6c)

Reply:
I'm not 100% sure why color_temp doesn't qualify so I added a request for a second opinion in case someone else knows

Reply:
Also some times it takes a few weeks/months to get a second opinion.  So we could always merge it without white support for now and you could add it in a future PR so we are only waiting for the second opinion on that PR